requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ pylint==2.6.0
 breathe==4.26.0
 torch==1.7.1
 sphinx-fortran==1.1.1
```

Comment:
We had talked about not making this a requirement, and instead having the individual conversion methods attempt importing xarray, with a catch of `ModuleNotFoundError` to discover that it's not available. I would have then expected a RedisRuntimeError to be created in that case that states that Xarray is needed.

Did the plans change?